app/Http/Transformers/ReactionTransformer.php

```diff
@@ -18,6 +18,15 @@ public function transform(Reaction $reaction)
         return [
             'northstar_id' => $reaction->northstar_id,
             'post_id' => (string) $reaction->post_id,
+            'term' => [
+                'name' => 'heart',
+                'id' => 641,
+                'total' => 1,
+            ],
+            'current_user' => [
```

Comment:
This information tells us if the current user (logged in user when Phoenix makes the request) has reacted to each of the posts that are being returned to populate the gallery. We then use this information to make the reaction button active or not active in the gallery.